pkg/apis/config/feature_flags_test.go

```diff
@@ -78,6 +79,16 @@ func TestNewFeatureFlagsFromConfigMap(t *testing.T) {
 			},
 			fileName: "feature-flags-bundles-and-custom-tasks",
 		},
+		{
+			expectedConfig: &config.FeatureFlags{
+				EnableAPIFields: "stable",
```

Comment:
Are we releasing this as a `stable` feature? or should this be behind `alpha` feature flag along with its own feature flag `enable-spire`?

Reply:
Sure. I agree that it should be behind the alpha flag for the time being. I will change that.